server-spi-private/src/main/java/org/keycloak/migration/ModelVersionUtils.java

```diff
@@ -0,0 +1,17 @@
+package org.keycloak.migration;
+
+public class ModelVersionUtils {
```

Comment:
I think this can be a method in `ModelVersion`. `ModelVersion.isSameMajorMinor(ModelVerison other)`

Reply:
done in next commit